src/projections/analysis/SumDetailReader.java

```diff
@@ -273,4 +449,32 @@ public int getNumIntervals() {
     protected List<RLEBlock>[] getData(int type) {
 	return rawData[type];
     }
+    	public double[] getMsg_count() 
+    {
+	return msg_count;
+    }
+
+	public double[] getMsg_size() 
+    {
+	return msg_size;
+    }
+	public double[] getMsg_recv_count() 
+    {
+	return msg_recv_count;
+    }
+
+	public double[] getMsg_recv_size() 
+    {
+	return msg_recv_size;
+    }
+	public double[] getMsg_recv_count_ext() 
+    {
+	return msg_recv_count_ext;
+    }
+
+	public double[] getMsg_recv_size_ext() 
+    {
+	return msg_recv_size_ext;
+    }

```

Comment:
Again, as stated earlier, these can use the existing `getData` method if they are read in as `RLEBlock`s.